lighthouse-core/report/report-generator.js

```diff
@@ -151,8 +151,10 @@ class ReportGenerator {
       }
 
       const partials = reportPartials.report.partials;
-      Handlebars.registerPartial(audit.name,
-        Handlebars.template(partials[audit.name] || partials['null-formatter']));
+      const partial = partials[audit.name] ||
+          partials[audit.extendedInfo.formatter] ||
```

Comment:
Cool. Kinda crazy that we are matching filenames against kebab case whereas formatter js files are hooked up directly. 
But w/e lets ship it.